utils/validators.utils.ts

```diff
@@ -0,0 +1,79 @@
+export const addressValidation = (e: any, setErrorState: (msg: string) => void) => {
+    const value = e.target.value;
+
+    if (value === 'acasa') {
+        setErrorState("Adresa nu poate fi casa ta!");
```

Comment:
Please rename all `setErrorState` methods to `setError`. The fact that we are referencing state should not be known to the validators.